build.gradle

```diff
@@ -232,6 +232,7 @@ allprojects {
         maven { url 'https://repo.gradle.org/gradle/libs-releases' }
         maven { url 'https://repo.gradle.org/gradle/libs-milestones' }
         maven { url "https://repo.gradle.org/gradle/libs-snapshots" }
+        maven { url "https://jitpack.io" }
```

Comment:
This will need to be removed as soon as possible: ideally snapshots should be published, or, depend on a release.